contents/blog/equity_explainer.md

```diff
@@ -0,0 +1,74 @@
+
+Usually, the first fundraising round (the ‘seed’ round) will involve carving out a chunk of the newly generated shares to create an ‘options pool’, which can be dipped into to provide options to newly hired employees as the company grows in size. 
+
+### What exactly is an option? 
+
+It's an option to purchase a share of the company at a given price – the “strike price” – which is known as “exercising” the option, thus becoming a percent owner of the company. However, these options are almost always granted by way of what is known as a vesting schedule, where options become available for exercise progressively over the course of employment with the company. This is to prevent a new hire from joining a company, exercising all of their options immediately, quitting, and watching their shares in the company grow in value without contributing any work towards building the company. 
+
+The most common vesting schedule for tech companies these days is the 4-year/1-year cliff, which means that a quarter of your options vest after your 1-year work anniversary, and 1/36th of your remaining options vest each month for the following three years. As a company, we feel that this structure strikes a fair balance between incentivizing team members to stay and contribute to the growth of the company, while not locking them into “golden handcuffs” where they are required to put in 4 years of work to reap any of the benefits of being a part of a start-up.
+
+### How do share options turn you into a millionaire? 
+
+When you are granted your options on the first day of employment, the price you will pay to exercise them will stay static at whatever the Fair Market Value (FMV) per share is on your first day, irrespective of what the value is when you sell your purchased shares. 
+
+The FMV is decided through an external, very complicated process known as a 409A, and the initial sale price of the share is usually set when the company "goes public" or "IPOs" by being listed on a public stock exchange like the NYSE. This is known as a “liquidation event” and in the simplest terms, the difference between your exercise price and the price of the share you sell in the public market is cash in your pocket. 
+
+The reality of this is much more complicated, as there can be a number of [tax implications](https://secfi.com/learn/exercise-stock-options-tax-implications) depending on the type of option you were granted, when you decide to exercise them relative to when you sell them, and your overall income; but you really should talk to an accountant before making any serious decisions so you don’t end up with an unexpectedly massive tax bill.
+
+Making matters more complicated, the majority of standard terms in stock option plans limit your freedom in when and how you can exercise your options. 
+
+## How PostHog does things differently
```

Comment:
It would be nice to start this section with a little more commentary on, philosophically, why we wanted to do things differently, before we get into practicalities of how we're different.

Reply:
(Jumping in as @jamesefhawkins @timgl and I implemented these terms before Grace joined)

There are a few justifications here:
- We had personally been burned by lame share option schemes when working at previous companies, so wanted to build something we wish we'd had. 
- It helps us to attract talent by making our share options massively more valuable in practical terms, not just the pure $ amount (which in many ways may be the least important thing). 
- It's part of our culture to expect high performance but reward people well and ensure they are genuinely able to share in PostHog's growth. Having a good share options scheme helps people feel more bought in. 
- (Lame but true) It's the right thing to do, dammit.